doc/ug/introduction.tex

```diff
@@ -321,6 +321,19 @@ \section{Requirements}
   version 1.2).
 \end{description}
 
+\subsection{Installing Requirements on Ubuntu 16.04 LTS}
+\label{sec:installing_requirements}
+\index{installing_requirements}
+
+To make ESPResSo run on Ubuntu 16.04 LTS its dependencies can be installed with:
+\begin{code}
+  apt install build-essential cmake cython python-numpy tcl-dev
+  tk-dev libboost-all-dev openmpi-common
```

Comment:
So did someone change the tcl finding routine in cmake, because that
at one point required tk, eventhough espresso wasn't using it.